src/html/angular-i18n-translations-extractor.ts

```diff
@@ -27,15 +43,13 @@ function handleTranslationsOfElements(
 ): void {
     const translationId = extractTranslationId(attribute, context);
 
-    let defaultTranslation: string;
     if (element.texts.length > 0 && translationId) {
-        defaultTranslation = element.texts[0].text;
         context.registerTranslation({
             translationId: translationId,
-            defaultText: defaultTranslation,
+            defaultText: element.texts[0].text,
             position: element.startPosition
         });
-    } else {
+    } else if (translationId) {
```

Comment:
Probably should be `!translationId`. But I would actually check for `element.texts.length` since this is what the message is about.